src/App.module.scss

```diff
@@ -1,38 +1,25 @@
-.App {
-  text-align: center;
-}
-
-.appLogo {
-  height: 40vmin;
-  pointer-events: none;
+.navbar ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: #333;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .appLogo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.navbar li {
+  float: left;
 }
 
-.appHeader {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
+.navbar li a {
+  display: block;
   color: white;
+  text-align: center;
+  padding: 14px 40px;
+  text-decoration: none;
 }
 
-.appLink {
-  color: #61dafb;
-}
-
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+/* Change the link color to #111 (black) on hover */
+.navbar li a:hover {
```

Comment:
we can make this clearer using Sass, and then probably don't need the comment any more

```scss
.navLink {
    float: left;
    transition: all 2s ease;

    &:hover {
        background-color: $backgroundHoverColor;
    }
}
```